metadata/release/sample.json

```diff
@@ -4,6 +4,22 @@
   "architectures": {
     "x86_64": {
       "media": {
+        "aliyun": {
+          "artifacts": {
+            "qcow2.xz": {
+              "disk": {
+                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-aliyun.qcow2.xz",
+                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190801.0/x86_64/fedora-coreos-30.20190801.0-aliyun.qcow2.xz.sig",
+                "sha256": "8f1492f1e9e94ec3f3ecef188c4a2da52348c4b830f6365181bd03e1d969f161"
+              }
+            }
+          },
+          "images": {
+            "us-east-1": {
+              "image": "ami-0506465824cb1b578"
```

Comment:
This is an entry for Amazon images nested under Aliyun; it looks like you have them swapped